lib/beaker-pe/install/pe_utils.rb

```diff
@@ -478,7 +478,7 @@ def verify_network_resources(hosts, network_resources)
             if network_resources != nil
               network_resources.each do |resource|
                 # curl the network resource silently (-s), only connect (-I), and don't print the output
-                on host, "curl -I -s #{resource} > /dev/null", :accept_all_exit_codes => true
+                on host, "curl -I -s --max-time 10 #{resource} > /dev/null", :accept_all_exit_codes => true
                 if host.connection.logger.last_result.exit_code != 0
                   logger.warn("Connection error: #{host.host_hash[:vmhostname]} was unable to connect to #{resource}. Please ensure that your test does not require this resource.")
```

Comment:
`host.host_hash[:vmhostname]` is the [same as `host[:vmhostname]`](https://github.com/puppetlabs/beaker/blob/master/lib/beaker/host.rb#L138-L140), and we would recommend using the latter rather than the former